### ISSUE TITLE
Change github user on atom

### ISSUE THREAD

**Issue by willmendil** [atom/github#2481]
## Summary

When connecting atom to github, I give a token and I'm all set. However, I have two github accounts and I don't understand how I can add a new account so I can switch which user/profil I'm working with.

## Motivation

https://i.stack.imgur.com/9fvxS.png
When I create a new repo, it asks you to select a user (the white box, there is indeed a ui issue there), but there is only one user. I have multiple github accounts, how do I add/change the user ?

## Describe alternatives you've considered

have two instances of atom installed, stable and nightly, or even use a vm for the second account (yes, rediculous i kinow)

## Additional context
-

**Comment by smashwilson** [comment]
Yeah, we don't have a great mechanism for this at present. You can _kind_ of do it by running `GitHub: Logout` from the command palette and going through the auth flow again, but it's a bit of a pain to make sure atom.io is generating a token with the right GitHub account on the re-login.

Also, we store your GitHub API token (which is tied to your GitHub account, and used for actions like creating repos) in your operating system's credential keychain, which makes flipping between different "profiles" harder 🤔

**Comment by willmendil** [comment]
Thanks for the response. I guess this should be a ne feature then. Will be installing viscode then ;)

**Comment by e-roy** [comment]
Hi @smashwilson, I'm using Windows and Atom, I've switched profiles, but I'm stuck on the operating system's credential keychain change.  I plan on staying with this account with Atom, so I won't be switching back and forth.  I'm a little green on this part of it, can you help give me a little more direction how I can make the change in the OS's credential keychain?

Thank you,

**Comment by finnfuchs** [comment]
@smashwilson It would be a really great feature to be able to switch GitHub accounts easily! I hope you can add that in the near future. Thanks for your great work.

**Comment by arathsin** [comment]
It would be a really great feature if I could do this on github.com as well (unrelated).

But yes, having to use seperate editors for personal vs work accounts is really a hassle.
